### PR TITLE
Clinic

### PR DESCRIPTION
**Pull request by tufoinnkuo10**
Added email column to owner table
insert data to visit tables
used Explain analyze to check runtime
added screenshots 
use pair programing
added pairs of screenshots before and after the index is added

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -67,6 +67,9 @@ CREATE TABLE visits (
 /*ALTER TABLE visits 
 ADD visit_date DATE;*/
 
+ALTER TABLE owners 
+ADD email varchar(120);
+
```
Comment (descholar-ceo) [inline schema.sql]: - [x] We were supposed to see the solution of this challenge in the schema.sql file 
  - [x] The expected solution is to add indexes. Adding indexes for FOREIGN KEYS or on columns frequently used in queries is one of the basic solutions for the performance problems so would you try it out.

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @tufoinnkuo10 and @VICTOR-CHUKWU , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._
- [x] You were supposed to add three pairs of screenshots in your PR description, which shows the `before` optimization and `after` optimization of your queries, 
  - [x] * The first screenshot should show how much time the query takes BEFORE the improvements introduced. - It the query planner output you should see the usage of `Seq scan` and/or `Filter` - The execution time should be around 1 sec (1000 ms).
  - [x] * The second screenshot should show how much time the query takes AFTER the improvements you introduced . - The execution time should be significantly lower than 1 sec (1000 ms).
  
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (tufoinnkuo10) [reply schema.sql]: thanks for the review.
we have made the required changes

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🎉🍾✔**
Hello @tufoinnkuo10 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**